### ISSUE TITLE
SSDP Discovery Not Working after Update to IOS 16.0

### ISSUE THREAD

**Issue by vunk93** [pierrickrouxel/SSDPClient#16]
This does not work anymore on iOS16 devices (but works on simulator).
Got this error : Encountered socket error:
The operation couldn’t be completed. 

[2023-05-31T14:45:42.421+07:00] [ERROR] [SSDPDiscovery.swift:132 discoverService(forDuration:searchTarget:port:)] Socket error: Error code: -9980(0x-26FC), No route to host
----> Error code: -9980(0x-26FC), No route to host
[2023-05-31T14:45:42.421+07:00] [ERROR] [SSDPDiscovery.swift:75 readResponses()] Socket error: Error code: -9982(0x-26FE), Bad file descriptor
----> Error code: -9982(0x-26FE), Bad file descriptor

Any ideas

**Comment by Chappi001** [comment]
I'm also facing the same issue. Have you found any fix for this?

**Comment by pierrickrouxel** [comment]
It's probably not a bug. See issue #13.

**Comment by Chappi001** [comment]
Oh, thanks @pierrickrouxel. Seems like I have to raise a request to apple support.

**Comment by nonegu** [comment]
This looks like resolved in following thread: https://developer.apple.com/forums/thread/715002

TL;DR iOS 15 wasn’t requiring the local network privacy entitlement when it should have been, with iOS 16 this has been fixed by Apple. So keep in mind to implement **_Multicast Entitlement_**.
